google-userinfo/background/main.js

```diff
@@ -0,0 +1,23 @@
+
+function notifyUser(user) {
+  browser.notifications.create({
+    "type": "basic",
+    "title": "Google info",
+    "message": `Hi ${user.name}`
+  });}
+
+function logError(error) {
+  console.error(`Error: ${error}`);
+}
+
+/*
+When the button's clicked:
+- get an access token using the identity API
+- use it to get the user's info
+- show a notification containing some of it
+*/
+browser.browserAction.onClicked.addListener(() => {
```

Comment:
I didn't make this change, because you actually get a pretty good diagnostic error message from Google if the client ID doesn't match. Also, I feel like this is not a check you'd want to have in your production WebExtension (it's only really relevant because this is a generic example) and I wouldn't want people copying it into their production code.